.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,54 @@
+name: Feature Request
+description: Suggest an idea for this project
+labels: "enhancement"
+body: 
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to request this feature!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe
+      description: Please provide a clear and concise description the problem this feature would solve. The more information you can provide here, the better.
+      placeholder: I'm always frustrated when...
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the proposed solution
+      description: Please provide a clear and concise description of what you would like to happen.
+      placeholder: I would like to see...
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: "Describe similar `asdf` features and why they are not sufficient"
+      description: "Describe asdf features you tried to use to accomplish what you wanted. Explain why they are insufficient for the task you were trying to accomplish."
+    validations:
+      required: true
+  - type: textarea
+    id: workarounds
+    attributes:
+      label: "Describe other workarounds you've considered"
+      description: "A clear and concise description of any alternative solutions or features you've considered."
+    validations:
+      required: true
+  - type: dropdown
+    id: importance
```

Comment:
Do you think this information will be valuable to us? A lot of users tend to think there feature request is more important than it really is.

Reply:
I agree most people do overestimate the importance. I think it would be nice to know whether the lack of the feature is stopping someone using `asdf` entirely or just a quality of life issue. Most quality of life issues we generally recommend people script around, but knowing the difference in the request upfront may be helpful.